README.md

```diff
@@ -26,16 +26,32 @@ source venv/bin/activate
 python -m pip install -r requirements.txt
 ```
 
+### Generate workflows
+
+Run the install script to generate the files to make the Github Actions work (from the root of your repo):
+
+```bash session
+python -m bench_runner install
+```
+
+This will create some files in `.github/workflows` as well as some configuration files at the root of your repo.
+Commit them to your repository, and push up to Github.
+
+```bash session
+git commit -a -m "Initial commit"
+git push origin main
+```
+
+The `bench_runner.toml` file created at the root of your repository contains configuration specific to your instance.
+More details about this configuration are below.
+
 ### Add some self-hosted runners
```

Comment:
Maybe I don't understand your comment... this is intended to be under "Generate Workflows", because the config file won't exist until after running `python -m bench_runner install`.

Reply:
Oh yes my bad. I left my previous comment because when I first tried the installation process, I've ran into the runtime error when there is no runner, like #225. So I thought we have to add a runner before running the installation command.

Reply:
I just tried locally to install bench_runner, I got this error:
```
ModuleNotFoundError: No module named 'tomllib'
```
And then I learned this module was added in 3.11. Should we add specific python version in requirements.txt?

Reply:
hmm even after I've upgraded my local python to 3.12, there is another error:
```
  File "/Users/melodywang009/Documents/Meta/github_projects/free-threading-benchmarking/venv3.12/lib/python3.12/site-packages/bench_runner/scripts/install.py", line 169, in generate_benchmark
    available_runners = [r for r in runners.get_runners() if r.available]
                                    ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/melodywang009/Documents/Meta/github_projects/free-threading-benchmarking/venv3.12/lib/python3.12/site-packages/bench_runner/runners.py", line 45, in get_runners
    conf = config.get_bench_runner_config().get("runners", [{}])[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/melodywang009/Documents/Meta/github_projects/free-threading-benchmarking/venv3.12/lib/python3.12/site-packages/bench_runner/config.py", line 14, in get_bench_runner_config
    with open(filepath, "rb") as fd:
         ^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'bench_runner.toml'
```
So seems like we need `bench_runner.toml` file before running `python -m bench_runner install`.

Reply:
> And then I learned this module was added in 3.11. Should we add specific python version in requirements.txt?

The pyproject.toml has that requirement in this PR (but admittedly, it's a new one, so it would require a re-install).

> So seems like we need bench_runner.toml file before running python -m bench_runner install.

Indeed.  Thanks for catching that.  I think we could address this by installing the template first.